src/Strategy/Go/GoModGraph.hs

```diff
@@ -117,44 +127,45 @@ parseGoModGraph = parseGoModPair `sepEndBy` (symbol "\n" <|> symbol "\r") <* eof
     parseGoModPair = (,) <$> parseGoGraphMod <*> parseGoGraphMod
 
 -- Builds graph from edges, main module, and selected module version set.
-buildGraph :: [(GoGraphMod, GoGraphMod)] -> GoGraphMod -> Set GoGraphMod -> Set GoGraphMod -> Bool -> Graphing.Graphing Dependency
-buildGraph fromToMods mainMod selectedMods directMods applyMVS =
-  Graphing.gmap toDependency
-    . Graphing.filter (withSelection)
+buildGraph :: [(GoGraphMod, GoGraphMod)] -> GoGraphMod -> Set GoGraphMod -> Set GoGraphMod -> Bool -> GoModReplacements -> Graphing.Graphing Dependency
```

Comment:
[Optional] Now that it has 7 args, let's simplify this. 

One suggestion that is quick/dirty is to use a container, it's not the ideal approach but would simplify and improve code readability. 

```haskell 
data GoGraphArgs = GoGraphArgs {
  edges :: [(GoGraphMod, GoGraphMod)],
  root :: GoGraphMod,
  ....
 . ...
}
```

Reply:
I noticed this as well and that might be good. One other option I thought of was trying to make wrapper types for each of the different kinds of GoGraphMod sets. At least then even with a function with so many arguments the type system would help make sure they're put in the right order. I will hack on that or try to make an argument record and push those changes.